server.js

```diff
@@ -28,7 +28,7 @@ const { MongoClient } = require('mongodb');
 const PORT = 80;
```

Comment:
The mongoUri variable is defined but not used. It can be removed to avoid confusion.